tools/docker-deploy.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+docker pull ubuntu:xenial
+echo -e "\nStarting container..."
+docker run -d --name allinone --privileged -v /sys/fs/cgroup:/sys/fs/cgroup:ro ubuntu:xenial systemd
+echo -e "\nUpdating APT cache..."
+docker exec -i allinone apt-get -qq update
+echo "Installing dependencies..."
+docker exec -i allinone apt-get -qq install python-apt ca-certificates apt-transport-https sudo ssh > /dev/null
+echo -e "\nRunning ansible deployment..."
+ansible-playbook -i inventory/allinone install-ci.yml tests/files/validate-ci.yml -c docker -vv -e @secrets.yml.example --skip-tags monitoring,not-on-docker,letsencrypt
```

Comment:
It would be better to allow the end user to pass along extra ansible arguments to this script, or split the ansible call out so that testing modifications to playbooks don't necessarily cause a full container build up to happen every time. I suppose those could be done as enhancements after the fact.

Maybe it would be better to simply document the ansible call in the Lore so that the developer has full control over the arguments used, and this script is simply just provisioning the targets.